src/WebViewTypes.ts

```diff
@@ -281,6 +289,27 @@ export interface IOSNativeWebViewProps extends CommonNativeWebViewProps {
   onContentProcessDidTerminate?: (event: WebViewTerminatedEvent) => void;
 }
 
+export interface MacOSNativeWebViewProps extends CommonNativeWebViewProps {
+  allowingReadAccessToURL?: string;
+  allowsBackForwardNavigationGestures?: boolean;
+  allowsInlineMediaPlayback?: boolean;
+  allowsLinkPreview?: boolean;
+  automaticallyAdjustContentInsets?: boolean;
+  bounces?: boolean;
+  contentInset?: ContentInsetProp;
+  contentInsetAdjustmentBehavior?:
+    | 'automatic'
+    | 'scrollableAxes'
+    | 'never'
+    | 'always';
```

Comment:
This would be nicer as a enum. So I can import these values and make sure I'm using the correct string.

Reply:
Sure.   But this is identical to iOS.   I can add an enum but I should make both the iOS and macOS conform to the same enum